tools/continuous_batching/benchmark/vlm_continuous_batching_benchmark.cpp

```diff
@@ -0,0 +1,413 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <fstream>
+#include <cstdlib>
+#include <chrono>
+#include <ostream>
+#include <random>
+#include <stdexcept>
+#include <thread>
+#include <mutex>
+#include <atomic>
+
+#include <nlohmann/json.hpp>
+#include <cxxopts.hpp>
+
+#include "openvino/genai/cache_eviction.hpp"
+#include "openvino/genai/tokenizer.hpp"
+#include "openvino/genai/continuous_batching_pipeline.hpp"
+#include "openvino/genai/generation_handle.hpp"
+#include "load_image.hpp"
+
+namespace {
+
+class AutoStartTimer {
+    const decltype(std::chrono::steady_clock::now()) m_start;
+public:
```

Comment:
Move common parts to a separate header in `tools/continuous_batching/benchmark/`